evm/vm/forks/sharding/vmc_utils.py

```diff
@@ -15,7 +20,7 @@ def get_vmc_json():
     return json.loads(vmc_json_str)
 
 
-def create_vmc_tx(TransactionClass, gasprice=GASPRICE):
+def create_vmc_tx(TransactionClass, gasprice=GAS_PRICE):
```

Comment:
two things.

- Naming nitpick to rename to `gas_price` for naming consistency
- Maybe just make `gasprice` required (remove the default value) to avoid the need to have this module be aware of config values.

Reply:
No problem. Will do.